### ISSUE TITLE
AndroidTV Accessibility: Movie Titles Unlabeled for Talkback Screen Reader

### ISSUE THREAD

**Issue by ertay** [Stremio/stremio-bugs#758]
**Describe the bug**
When using the Talkback screen reader on the Android TV app, on the home page where you can browse the rows of movies or shows, the titles of the movies and shows are not labeled. So, as a blind user, I do not know which item I am currently focused on. To see the title name, I need to enter the details and then look for the name, if it is the wrong one, I need to go back and try again with a different unlabeled title in the row. On the web version of the app, the movie titles are shown as hyperlinks, and they can be more easily browsed.
This problem also exists when you search for titles. The search results are all unlabeled, so when you focus the items with Talkback, you are just told that the item is unlabeled.

**To Reproduce**
1. Enable Talkback from Accessibility settings.
2. Launch Stremio and in the home page try to go through your popular movies or series rows. Notice that when you focus a specific title Talkback does not read the name of the title, instead it says Unlabeled.
3. Go to Search, search for a title, and then inspect the search results. Notice that the results are also unlabeled.

**Expected behavior**
When titles are focused with talkback on the home page, see all view, or search results, Talkback should be reading the full name of the title in focus, and it should not say 'Unlabeled'.

**TV (please complete the following information):**
 - Device: Nvidia shield TV Pro 2019
 - OS: Android TV (Android 9)
 - Version 1.6.9
